packages/react/src/migrations/update-19-6-0/update-ssr-server-port.ts

```diff
@@ -0,0 +1,132 @@
+import {
+  ProjectConfiguration,
+  type Tree,
+  getProjects,
+  joinPathFragments,
+  visitNotIgnoredFiles,
+} from '@nx/devkit';
+import { tsquery } from '@phenomnomnominal/tsquery';
+import * as ts from 'typescript';
+import { minimatch } from 'minimatch';
+
+export default function update(tree: Tree) {
+  const projects = getProjects(tree);
+  const executors = [
+    '@nx/webpack:ssr-dev-server',
+    '@nx/react:module-federation-ssr-dev-server',
+  ];
+
+  for (const project of projects.values()) {
```

Comment:
It'd be easier to use `forEachExecutorOptions` here?

Reply:
yup will use that instead